bridge-fungible-token/src/utils.sw

```diff
@@ -1,11 +1,187 @@
 library utils;
 
-use std::mem::read;
+dep errors;
+dep events;
+dep data;
+
+use std::{
+    alloc::alloc,
+    mem::copy,
+    outputs::{
+        Output,
+        output_count,
+        output_type,
+    },
+    revert::revert,
+    vec::Vec,
+};
+
+use errors::BridgeFungibleTokenError;
+use events::{BurnEvent, MintEvent, TransferEvent};
+use data::MessageData;
+use std::{
+    constants::ZERO_B256,
+    inputs::{
+        Input,
+        input_pointer,
+        input_type,
+    },
+    logging::log,
+    mem::{
+        addr_of,
+        read,
+        write,
+    },
+    token::{
+        burn,
+        mint_to_address,
+        transfer_to_address,
+    },
+    vm::evm::evm_address::EvmAddress,
+};
 
 // TODO: [std-lib] remove once standard library functions have been added
 const GTF_INPUT_MESSAGE_DATA_LENGTH = 0x11B;
 const GTF_INPUT_MESSAGE_DATA = 0x11E;
+const GTF_INPUT_MESSAGE_SENDER = 0x115;
+const GTF_INPUT_MESSAGE_RECIPIENT = 0x116;
+
+pub fn mint_and_transfer_tokens(amount: u64, to: Address) {
+    mint_to_address(amount, to);
+    log(MintEvent { amount, to });
+}
+
+pub fn burn_tokens(amount: u64, from: Identity) {
```

Comment:
these mint and burn functions are a little small to be helpful as an abstracted util, especially if we remove logging for simple mint and burn as brought in previous comment

Reply:
I'll revamp the events in the contract and then reassess these utils, same for `transfer_tokens` below.